src/test/java/teammates/test/cases/action/InstructorFeedbackEditCopyActionTest.java

```diff
@@ -413,6 +415,14 @@ protected InstructorFeedbackEditCopyAction getAction(String... params) {
     @Override
     @Test
     protected void testAccessControl() throws Exception {
-        //TODO: implement this
+        String[] params = new String[]{
+                Const.ParamsNames.FEEDBACK_SESSION_NAME, "First feedback session",
+                Const.ParamsNames.COURSE_ID, "idOfTypicalCourse1",
+                Const.ParamsNames.COPIED_FEEDBACK_SESSION_NAME, "Session with valid name",
+                Const.ParamsNames.COPIED_COURSES_ID, "idOfTypicalCourse2",
+                Const.ParamsNames.COPIED_COURSES_ID, "idOfSampleCourse-demo"
+        };
+
+        verifyUnaccessibleWithoutModifyCoursePrivilege(params);
```

Comment:
InstructorFeedbackEditCopyAction checks for `INSTRUCTOR_PERMISSION_MODIFY_SESSION`, so should use `verifyUnaccessibleWithoutModifySessionPrivilege`.